app-crypt/libu2f_host/libu2f_host-1.1.10.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Yubico Universal 2nd Factor (U2F) Host C Library"
+DESCRIPTION="libu2fhost provides a C library and command-line tool \
+that implements the host-side of the U2F protocol. There are APIs to \
+talk to a U2F device and perform the U2F Register and U2F Authenticate \
+operations."
+HOMEPAGE="https://developers.yubico.com/libu2f-host/
+		https://github.com/Yubico/libu2f-host"
+COPYRIGHT="2013-2014 Yubico AB"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Yubico/libu2f-host/archive/libu2f-host-$portVersion.tar.gz"
+CHECKSUM_SHA256="45937c6c04349f865d9f047d3a68cc50ea24e9085d18ac2c7d31fa38eb749303"
+SOURCE_DIR="libu2f-host-libu2f-host-$portVersion"
+PATCHES="libu2f_host-1.1.10.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libu2f_host = $portVersion
```

Comment:
This can't be build for x86_gcc2, so add $secondaryArchSuffix here and in other places related